### PR TITLE
Use Markdown for Sections 1-3

### PR DESCRIPTION
**Pull request by odejesush**
This change updates the spec source file to use more Markdown syntax in
order to improve the readability. This commit updates the spec from
Section 1 to Section 3.

The following changes were performed:
* Heading tags were replaced with their Markdown equivalent, #.
* \<p\> tags were removed.
* \<ol\>, \<ul\>, and \<li\> tags were replaced with their Markdown equivalent, * and 1.
* \<table\> CSS was updated to match the tables in the WebUSB spec so that
they didn't overflow past the containers that were holding them.

***
<a href="https://pr-preview.s3.amazonaws.com/odejesush/web-bluetooth/pull/469.html" title="Last updated on Feb 11, 2020, 9:58 PM UTC (55cdebe)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/WebBluetoothCG/web-bluetooth/469/4ca7ccb...odejesush:55cdebe.html" title="Last updated on Feb 11, 2020, 9:58 PM UTC (55cdebe)">Diff</a>

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -145,2035 +145,1682 @@ spec: webidl
...
+        calls threw an exception, throw that exception and abort these steps.
+    1. Remove from <var>optionalServiceUUIDs</var> any UUIDs that are
+        <a>blocklisted</a>.
+1. Let |descriptor| be
+    <xmp highlight="js">
+    {
+      name: "bluetooth",
+      filters: |uuidFilters|,
+      optionalServices: |optionalServiceUUIDs|,
+      acceptAllDevices: |filters| !== null,
```
Comment (reillyeon) [inline index.bs]: The `|` characters are not being interpreted as Markdown.

File: index.bs [hunk 2]
```diff
@@ -145,2035 +145,1682 @@ spec: webidl
...
-    <em>This section is non-normative.</em>
+    <div class="note">
+      Note: The UA should show the user the human-readable name of each device.
+      If this name is not available because, for example, the UA's Bluetooth
+      system doesn't support privacy-enabled scans, the UA should allow the user
+      to indicate interest and then perform a privacy-disabled scan to retrieve
+      the name.
+    </div>
+1. If |device| is {{"denied"}}, return `[]` and abort these steps.
+1. <div class="note">
```
Comment (reillyeon) [inline index.bs]: This note shouldn't be a step.

File: index.bs (hunk 1 above)
Comment (odejesush) [reply index.bs]: Thanks.

File: index.bs (hunk 2 above)
Comment (odejesush) [reply index.bs]: I agree. I'm not sure why it was a step in the first place, but I moved it into step 9 of this algorithm where it says that the UA should prompt the user to choose. I placed it below the other note about showing human-readable names.

**Comment by odejesush** [comment]
Thanks Reilly. The force push was to fix the commit history of my forked repo branch after attempting to update with the User Activation v2 change.

**Pull request merged**